scripts/globals/mobskills/target_analysis.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------
+-- Target Analysis
+--
+-- Description: AoE Absorb All with randomness
+-- Type: Magical
+---------------------------------------------
+require("scripts/globals/mobskills")
+require("scripts/globals/status")
+require("scripts/settings/main")
+local ID = require("scripts/zones/Sealions_Den/IDs")
+local ID2 = require("scripts/zones/Apollyon/IDs")
+---------------------------------------------
+local mobskill_object = {}
+
+local attributes =
+{
+    xi.effect.STR_DOWN,
+    xi.effect.DEX_DOWN,
+    xi.effect.VIT_DOWN,
+    xi.effect.AGI_DOWN,
+    xi.effect.MND_DOWN,
+    xi.effect.INT_DOWN,
+    xi.effect.CHR_DOWN,
+}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    local mobhp = mob:getHPP()
+    local id = mob:getID()
+    local checkID = false
```

Comment:
This seems a bit unclear (and looking at the loop), what is the purpose of this?